src/consume.ts

```diff
@@ -830,9 +832,38 @@ function messageViewerStartPollingCommand(
     handler.dispose();
     os.dispose();
   });
-}
 
-const logger = new Logger("consume");
+  type TrackAction = {
```

Comment:
Is the type for the parameter to `track()` and `debouncedTrack()`. Was just `object` before, but all calls were at least providing `action`. If / when want to support additional fields, or open-ended fields but still also `action`, can respell accordingly.